Constrained_triangulation_3/examples/Constrained_triangulation_3/remesh_constrained_Delaunay_triangulation_3.cpp

```diff
@@ -42,8 +42,6 @@ using Constraints_pmap = CGAL::Boolean_property_map<Constraints_set>;
 int main(int argc, char* argv[])
 {
   std::string filename = (argc > 1) ? argv[1] : CGAL::data_file_path("meshes/mpi.off");
-  double target_edge_length = (argc > 2) ? std::stod(argv[2]) : 1.0;
```

Comment:
Why do the command line parameters get removed.  Later we see a hard coded target edge length of `1.`  but that may make no sense in case another input than `mpi.off`  is given.